.github/opened-pr-labeler.yml

```diff
@@ -0,0 +1,5 @@
+# https://github.com/actions/labeler
+
+# Add 'Status: Do Not Merge' label to any newly opened PR
+"Status: Do Not Merge":
```

Comment:
How do we change the labels on PR's to remove "Do Not Merge"?

Reply:
You just remove the label from the right hand side of the PR page.